doc/stdenv/platform-notes.chapter.md

```diff
@@ -46,22 +46,195 @@ Some common issues when packaging software for Darwin:
   }
   ```
 
-- Some packages assume xcode is available and use `xcrun` to resolve build tools like `clang`, etc. This causes errors like `xcode-select: error: no developer tools were found at '/Applications/Xcode.app'` while the build doesn’t actually depend on xcode.
+- Some packages assume Xcode is available and use `xcrun` to resolve build tools like `clang`, etc. The Darwin stdenv includes `xcrun`, and it will return the path to any binary available in a build.
 
   ```nix
   stdenv.mkDerivation {
     name = "libfoo-1.2.3";
     # ...
+    nativeBuildInputs = [ bison ];
+    buildCommand = ''
+      xcrun bison foo.y # produces foo.tab.c
+      # ...
+    '';
+  }
+  ```
```

Comment:
Probably we could just drop all of this, right? It’s something users no longer really need to know about.

Reply:
Given that the previous recommendation was to modify the build process to remove uses of `xcrun`, I think it’s important to call out that that is no longer necessary to do that. For those who want to argue that it is, the documentation provides an authoritative answer that it is now.

Reply:
Fair enough – maybe that could go in the migration section?